tests/__init__.py

```diff
@@ -1,3 +1,3 @@
-from mlflow.utils.logging_utils import _configure_mlflow_loggers
+#from mlflow.utils.logging_utils import _configure_mlflow_loggers
```

Comment:
backed out this accidental change change in `7018d2473aacb95290028bf0f7546c706c0acb5a`